src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -10,17 +10,23 @@
     <RootNamespace>System.Runtime.Serialization.Json.Tests</RootNamespace>
     <AssemblyName>System.Runtime.Serialization.Json.Tests</AssemblyName>
     <ProjectGuid>{701CB3BC-00DC-435D-BDE4-C5FC29A708A7}</ProjectGuid>
+    <UseProjectReferences>true</UseProjectReferences>
+  </PropertyGroup>
+  <PropertyGroup Condition="!$(UseProjectReferences)" >
```

Comment:
By default this should be true so if we make any changes to the product we can run tests against the changes. 
The false option is for when we want to run performance tests against Desktop facades. We set this to false so the tests compile with contracts and will run against Desktop as well, otherwise it will fail because there is not private implementation on Desktop. I think this scenario doesn't happen frequently.